ACKategories/UIApplicationExtensions.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  UIApplicationExtensions.swift
+//  ACKategories
+//
+//  Created by Igor Rosocha on 12/16/19.
+//  Copyright © 2019 Ackee, s.r.o. All rights reserved.
+//
+
+import UIKit
+
+public extension UIApplication {
+    /// Completely clear launch screen cache
+    func clearLaunchScreenCache() {
+        do {
+            try FileManager.default.removeItem(atPath: NSHomeDirectory()+"/Library/SplashBoard")
```

Comment:
Please add spaces around the plus `+` sign 🙂

Reply:
Definitely 🙂